lib/definitions.php

```diff
@@ -760,6 +762,110 @@
 	EXISTINGASSIGNMENT_DELETE => trans('<!existingassignment>delete'),
 );
 
+//Request Tracker Ticket Types
+define('RTT_OFFER', 1);
```

Comment:
Jeśli mielibyśmy się trzymać zasad stosowanych dotąd to powinno być:
RT_TYPE_OFFER
RT_TYPE_DOCS
RT_TYPE_INST
RT_TYPE_MOD
...
itd.